addons/mail/static/src/components/rtc_invitation_card/rtc_invitation_card.xml

```diff
@@ -3,29 +3,29 @@
 
     <t t-name="mail.RtcInvitationCard" owl="1">
         <t t-if="rtcInvitationCard">
-            <div class="o_RtcInvitationCard" t-attf-class="{{ className }}" t-ref="root">
+            <div class="o_RtcInvitationCard d-flex flex-column m-2 p-5 border border-dark rounded-sm bg-900" t-attf-class="{{ className }}" t-ref="root">
                 <t t-if="rtcInvitationCard.thread.rtcInvitingSession">
-                    <div class="o_RtcInvitationCard_partnerInfo">
-                        <img class="o_RtcInvitationCard_partnerInfoImage rounded-circle"
+                    <div class="o_RtcInvitationCard_partnerInfo d-flex flex-column justify-content-around align-items-center text-nowrap">
+                        <img class="o_RtcInvitationCard_partnerInfoImage mb-2 rounded-circle o_cursor_pointer"
                             t-att-src="rtcInvitationCard.thread.rtcInvitingSession.avatarUrl"
                             t-on-click="rtcInvitationCard.onClickAvatar"
                             alt="Avatar"/>
-                        <span class="o_RtcInvitationCard_partnerInfoName" t-esc="rtcInvitationCard.thread.rtcInvitingSession.name"/>
-                        <span class="o_RtcInvitationCard_partnerInfoText">Incoming Call...</span>
+                        <span class="o_RtcInvitationCard_partnerInfoName w-100 overflow-hidden font-weight-bolder text-truncate text-center" t-esc="rtcInvitationCard.thread.rtcInvitingSession.name"/>
+                        <span class="o_RtcInvitationCard_partnerInfoText font-italic opacity-75">Incoming Call...</span>
                     </div>
                 </t>
-                <div class="o_RtcInvitationCard_buttonList">
-                    <div class="o_RtcInvitationCard_buttonListButton o_RtcInvitationCard_buttonListRefuse"
+                <div class="o_RtcInvitationCard_buttonList d-flex w-100 justify-content-around align-items-center mt-4">
+                    <div class="o_RtcInvitationCard_buttonListButton o_RtcInvitationCard_buttonListRefuse p-2 rounded-circle border-0 bg-danger o_cursor_pointer o_user_select_none"
                         aria-label="Refuse"
                         title="Refuse"
                         t-on-click="rtcInvitationCard.onClickRefuse">
-                        <i class="o_RtcInvitationCard_buttonListButtonIcon fa fa-lg fa-times"/>
+                        <i class="o_RtcInvitationCard_buttonListButtonIcon fa fa-lg fa-times m-3"/>
                     </div>
-                    <div class="o_RtcInvitationCard_buttonListButton o_RtcInvitationCard_buttonListAccept"
+                    <div class="o_RtcInvitationCard_buttonListButton o_RtcInvitationCard_buttonListAccept p-2 rounded-circle border-0 bg-success o_cursor_pointer o_user_select_none"
```

Comment:
Since you replaced the css by a `bg-success` on a div, you lost the hover state. To keep it, you could probably use a `<button/>` instead of the `<div/>` (same with `bg-danger`).
To be tested.